docs/ops/detection/ExperimentalDetectronGenerateProposalsSingleImage_8.md

```diff
@@ -0,0 +1,156 @@
+5.  Takes top *pre_nms_count* proposals, if total number of proposals is less than *pre_nms_count* takes all proposals.
+6.  Applies non-maximum suppression with *adaptive_nms_threshold*. The initial value of *adaptive_nms_threshold* is
+*nms_threshold*. If `nms_eta < 1` and `adaptive_threshold > 0.5`, update `adaptive_threshold *= nms_eta`.
+7.  Takes top *post_nms_count* proposals and returns these top proposals, scores and the number of proposals. If
+attribute *dynamic_output* is false and total number of proposals is less than *post_nms_count* returns output tensors
+filled with zeroes. If *dynamic_output* is false, then need not to fill output tensors with zeros. No matter attribute
+*dynamic_output* is true or false, the value will be the actual number of proposals.
+
+**Attributes**:
+
+* *min_size*
+
+    * **Description**: The *min_size* attribute specifies minimum box width and height.
+    * **Range of values**: non-negative floating-point number
+    * **Type**: float
+    * **Default value**: None
+    * **Required**: *yes*
+
+* *nms_threshold*
+
+    * **Description**: The *nms_threshold* attribute specifies threshold to be used in the NMS stage.
+    * **Range of values**: non-negative floating-point number
+    * **Type**: float
+    * **Default value**: None
+    * **Required**: *yes*
+
+* *pre_nms_count*
+
+    * **Description**: The *pre_nms_count* attribute specifies number of top-n proposals before NMS.
+    * **Range of values**: non-negative integer number
+    * **Type**: int
+    * **Default value**: None
+    * **Required**: *yes*
+
+* *post_nms_count*
+
+    * **Description**: The *post_nms_count* attribute specifies number of top-n proposals after NMS.
+    * **Range of values**: non-negative integer number
+    * **Type**: int
+    * **Default value**: None
+    * **Required**: *yes*
+
+* *coordinates_offset*
+
+    * **Description**: The *coordinates_offset* attribute specifies the relationship between ROI location points and ROI width and height. For example if *coordinates_offset* is false, the `width = x_right - x_left`. If *coordinates_offset* is true, the `width = x_right - x_left + 1`.
+    * **Range of values**: true or false
+    * **Type**: boolean
+    * **Default value**: false
+    * **Required**: *no*
+
+* *nms_eta*
+
+    * **Description**: eta parameter for adaptive NMS.
+    * **Range of values**: a floating-point number in close range `[0, 1.0]`.
+    * **Type**: float
+    * **Default value**: `1.0`
+    * **Required**: *no*
+
+* *dynamic_output*
+
+    * **Description**: The *dynamic_output* attribute specifies whether the output ROIs and ROIs scores are dynamic.
+    * **Range of values**: true or false
+    * **Type**: boolean
+    * **Default value**: false
+    * **Required**: *no*
+
+
+**Inputs**
+
+* **1**: A 1D tensor of type *T* with 3 elements `[image_height, image_width, scale_height_and_width]` providing input
```

Comment:
The open here - how to make "im_shape" and "im_info" compatiable?

 V1 uses "im_info", which is image_height, image_width, and scale_height_and_width.
  
V2 should be "im_shape", which is input_height, input_width. There will be another parameter of model called "scale_factor". It is scale_width, and scale_height.
If scale_width != scale_height, there is no way to convert "im_shape" to "im_info".

Reply:
V2 is {width, height}. Just need to convert to {width, height, 1} and input it to experimental generate proposal.
But seems the scale doesn't work in experimental generate proposal, it just have this interface but have no implementation.